public/app/features/playlist/api.ts

```diff
@@ -38,7 +38,7 @@ class LegacyAPI implements PlaylistAPI {
 }
 
 interface K8sPlaylistList {
-  playlists: K8sPlaylist[];
+  items: K8sPlaylist[];
```

Comment:
same fix in https://github.com/grafana/grafana/pull/75709/files#diff-e61e223f2faff75e3022c9d1ea535fdeb789ff5bf903cc55d44943bd246133f9R41

Reply:
removed from this PR